migrate.py

```diff
@@ -20,6 +20,8 @@
 #######################
 # CONFIG SECTION START
 #######################
+# Gitlab user to use as a fallback for missing members for groups
+GITLAB_FALLBACK_USERNAME = os.getenv('GITLAB_FALLBACK_USERNAME', 'gitea_admin')
```

Comment:
can we call this variable something like GITEA_FALLBACK_GROUP_MEMBER ? The current name is a little bit missleading

Reply:
👍